problems/basic_test.go

```diff
@@ -0,0 +1,42 @@
+package problems
+
+import (
+	"context"
+	"encoding/binary"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.opencensus.io/trace"
```

Comment:
Maybe alias these imports like in `basic.go` so it's more explicit when opencensus vs datadog is used